easybuild/easyblocks/i/imkl.py

```diff
@@ -112,7 +112,9 @@ def make_module_req_guess(self):
                     'FPATH': ['mkl/include', 'mkl/include/fftw'],
                 }
                 if LooseVersion(self.version) >= LooseVersion('11.0'):
-                    if LooseVersion(self.version) >= LooseVersion('11.1'):
+                    if LooseVersion(self.version) >= LooseVersion('11.3'):
+                        retdict['MIC_LD_LIBRARY_PATH'] = ['lib/intel64_lin_mic', 'mkl/lib/mic'];
```

Comment:
I think it does not matter...the easyblock already references intel64 in several places which is actually a symlink to intel64_lin